### PR TITLE
feat(changelog): billing-added-discover-our-cost-manager-tool 2024-03-28

### PR DESCRIPTION
**Pull request by ofranc**
Reporter: Julie PATUREL

This PR is a new changelog contribution. It has been created automatically.

Make sure to review it carefully before merging it.

### CODE REVIEW

File: changelog/march2024/2024-03-28-billing-added-discover-our-cost-manager-tool.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+status: added
+author:
+    fullname: 'Join the #billing-cost-management channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2024-03-28
+category: billing
+product: billing
+---
+
+Introducing Cost Manager tool:
```
Comment (bene2k1) [inline changelog/march2024/2024-03-28-billing-added-discover-our-cost-manager-tool.mdx:12]:
```suggestion
We are introducing our Cost Manager tool, that helps you to:
```

File: changelog/march2024/2024-03-28-billing-added-discover-our-cost-manager-tool.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Simplifies tracking of cloud resource costs.
```
Comment (bene2k1) [inline changelog/march2024/2024-03-28-billing-added-discover-our-cost-manager-tool.mdx:14]:
```suggestion
- Simplify tracking of cloud resource costs.
```

File: changelog/march2024/2024-03-28-billing-added-discover-our-cost-manager-tool.mdx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+- Access 5 years of data for better analysis.
+- Effortlessly monitor costs with customizable filters.
+- Utilize the billing API for seamless data retrieval.
```
Comment (bene2k1) [inline changelog/march2024/2024-03-28-billing-added-discover-our-cost-manager-tool.mdx:17]:
```suggestion
- Use the billing API for seamless data retrieval.
```

File: changelog/march2024/2024-03-28-billing-added-discover-our-cost-manager-tool.mdx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+Ready to try your hand at enhancing your cloud cost management?
```
Comment (bene2k1) [inline changelog/march2024/2024-03-28-billing-added-discover-our-cost-manager-tool.mdx:19]:
```suggestion
Discover the Cost Manager tool in the billing section of your [Scaleway console](https://console.scaleway.com/billing/consumption).
```

**Review by bene2k1 — APPROVED** [review]

File: changelog/march2024/2024-03-28-billing-added-discover-our-cost-manager-tool.mdx (hunk 4 above)
Comment (nerda-codes) [reply changelog/march2024/2024-03-28-billing-added-discover-our-cost-manager-tool.mdx:19]: Maybe we can also link to Luiza's doc?

**Review by ldecarvalho-doc — APPROVED** [review]

File: changelog/march2024/2024-03-28-billing-added-discover-our-cost-manager-tool.mdx (hunk 4 above)
Comment (bene2k1) [reply changelog/march2024/2024-03-28-billing-added-discover-our-cost-manager-tool.mdx]: wdyt @nerda-codes ?

**Pull request merged**